src/bootstrap.jl

```diff
@@ -544,40 +549,89 @@ function tidyσs(bsamp::MixedModelFitCollection{T}) where {T}
     return result
 end
 
+_nρ(d::Diagonal) = 0
+_nρ(t::LowerTriangular) = kchoose2(size(t.data, 1))
+
+function σρnms(λ)
+    σsyms = _generatesyms('σ', sum(first ∘ size, λ))
+    ρsyms = _generatesyms('ρ', sum(_nρ, λ))
+    val = Symbol[]
```

Comment:
```suggestion
    val = Symbol[]
    sizehint!(val, length(σsyms) + length(ρsyms))
```

Reply:
My feeling was that the call to `sizehint!` might take more time than any resizing that needs to be done but it shouldn't be a big consideration.

Reply:
Yeah, ideally we're talking about relatively small arrays, but then again somebody will try to fit a maximal model with 100+ covariance parameters.... :thinking: